packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1342,6 +1354,16 @@ class TextInputConnection {
     TextInput._instance._show();
   }
 
+  /// Start Live Text input.
+  ///
+  /// This method should only be called when Live Text input is available.
+  /// See also:
+  ///  * [LiveText], where we can get the the availability of Live Text input.
```

Comment:
From the style guide, avoid using words like "we".  Maybe something like this instead: "where the availability of Live Text input can be obtained."